demo/README.md

```diff
@@ -39,6 +38,14 @@ From the root of this repository execute:
 ./demo/run-osm-demo.sh
 ```
 
+### OpenShift
+To run the demo on OpenShift, you must enable privileged init containers.
+```shell
+./demo/run-osm-demo.sh --set="OpenServiceMesh.enablePrivilegedInitContainer=true"
+```
```

Comment:
I considered that early on but it didn't seem compatible if users also passed in additional arguments.

For example, this doesn't work: 
```
bin/osm install --set="OpenServiceMesh.enablePrivilegedInitContainer=true" --set="OpenServiceMesh.envoyLogLevel=info"
```

Reply:
In general, using `--set` multiple times should work. In this case, `--set="OpenServiceMesh.envoyLogLevel=info"` doesn't work because the `--envoy-log-level` flag exists.

Reply:
Thanks for figuring that out! Seems like a bug to me so I filed https://github.com/openservicemesh/osm/issues/2806

The latest commit enables privileged init container when DEPLOY_ON_OPENSHIFT is set to true.

Reply:
If the script is setting this value automatically, should we remove this?

Reply:
removed